test/gst-msdk/util.py

```diff
@@ -32,6 +32,7 @@ def get_supported_format_map():
     "P010"  : ("p010-10le", "P010_10LE"),
     "P210"  : ("i422-10le", "I422_10LE"),
     "P410"  : ("y444-10le", "Y444_10LE"),
+    "Y410"  : ("Y410", "Y410"),
```

Comment:
The first entry in the tuple should be lowercase

Reply:
To be clear, the first entry is used in elements with GstVideoFormat properties (e.g. vaapipostproc).  The second entry is for capsfilter.  I guess we should add a comment in the code about this.

Reply:
ok, I will

Reply:
updated